source/_integrations/ituran.markdown

```diff
@@ -0,0 +1,48 @@
+---
+title: Ituran
+description: Instructions on how to add Ituran to Home Assistant.
+ha_category:
+  - Car
+  - Device Tracker
+  - Presence detection
+ha_release: '2025.1'
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@shmuelzon'
+ha_domain: ituran
+ha_platforms:
+  - device_tracker
+ha_integration_type: integration
+---
+
+The **Ituran** {% term integration %} allows you to retrieve information from your Ituran-equipped vehicle. It pulls information from the Ituran web service regarding the vehicle's location.
+
+{% note %}
+The information is pulled every 5 minutes from the Ituran web service; however, the data is updated at intervals determined by Ituran based on whether the vehicle is stationary or not.
+{% endnote %}
```

Comment:
We don't have to add a big note in the top for information like this.

Instead, display it as part of the documentation below, here is an example as documented in our developer documentation: <https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-data-update>

Reply:
Done. Also moved it to be below the prerequisites, seemed like a better flow 🤷‍♂️